### PR TITLE
Revise Versioning Structure v3.1.12 --> v3.1

### PR DESCRIPTION
**Pull request by nate-double-u**
Moving files.

Updating config.toml content/docs/v3.1/_index.md layouts/index.redirects layouts/partials/docs/hero.html layouts/partials/docs/nav-panel.html to suit.

Contributes to: https://github.com/etcd-io/website/issues/152

/cc @chalin

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy previews:
* https://deploy-preview-156--etcd.netlify.app/docs/current/
* https://deploy-preview-156--etcd.netlify.app/docs/v3.1.12
* https://deploy-preview-156--etcd.netlify.app/docs/v3.1.12/
* http://deploy-preview-156--etcd.netlify.app/docs/v3.1/libraries-and-tools/ (Libraries & tools page was dropped after v3.1.12, so is  a good page to use to verify the behaviour of the version selectors).

File: layouts/partials/docs/hero.html [hunk 1]
```diff
@@ -73,7 +73,7 @@
               </span>
               {{ end }}
             </a>
-            {{ else if ne "/v2.3/" $new_version }}
+            {{ else if ((ne "/v2.3/" $new_version) | and (ne "/v3.1/" $new_version)) }}
```
Comment (chalin) [inline layouts/partials/docs/hero.html:76]: Maybe in the next round of changes we could you a regex-match test here (and in the nav-panel.html file)?

Comment (nate-double-u) [reply layouts/partials/docs/hero.html:76]: That's sortof how i'm doing it in the `docsy` branch.

Comment (nate-double-u) [reply layouts/partials/docs/hero.html:76]: https://github.com/etcd-io/website/blob/bbc8bf14e3328400e00c75f94737594b5b434144/layouts/partials/navbar-version-selector.html#L22-L38

**Review by chalin — APPROVED** [review]

**Pull request merged**